packages/types/src/Metadata/v3/toV2.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright 2017-2019 @polkadot/types authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import MetadataV2 from '../v2/Metadata';
+import MetadataV3 from '.';
+
+export default function toV2 (v2: MetadataV3): MetadataV2 {
+  return new MetadataV2(v2);
```

Comment:
`DoubleMapType` only exists in v3 so this should fail if trying to convert a metadata includes double to v2.
One way to solve this (without refactor everything to make upgrade instead of downgrade), is duplicate `DoubleMapType` in v2 like how I retrofit `is_linked` in v1